documentation/integrations/astro.md

```diff
@@ -37,7 +43,8 @@ import { ScalarComponent } from '@scalar/astro'
 
 <ScalarComponent configuration={{
   url: '/openapi.json',
-  theme: 'purple',
+  pageTitle: 'Awesome API',
+  cdn: 'https://cdn.jsdelivr.net/npm/@scalar/api-reference@latest',
```

Comment:
Is `pageTitle` part of Scalar configuration object (going to be introduced), or a specific property for Astro integration?